components/studio/libexec/hab-studio-type-default.sh

```diff
@@ -119,6 +119,7 @@ sup-run() {
   echo "--> Launching the Habitat Supervisor in the background..."
   echo "    Running: hab sup run \$*"
   hab sup run \$* > /hab/sup/default/sup.log &
+  echo \$! > /hab/sup/default/launch.pid
```

Comment:
Super clean, I like that we didn't modify the launcher/supervisor code at all and instead wrote the pid in the studio for it's purposes